Essentials/src/config.yml

```diff
@@ -319,9 +319,17 @@ skip-used-one-time-kits-from-kit-list: false
 # Determines the functionality of the /createkit command.
 # If this is true, /createkit will give the user a link with the kit code.
 # If this is false, /createkit will add the kit to the kits.yml config file directly.
-#
+# Default is false.
 pastebin-createkit: false
 
+# Determines if /createkit will generate kits with item serialization.
+# By using serialization, there is added support for for complex items such as potions and shulker boxes.
+# However, serialization removes the ability to edit kits by hand and kits *may* break in future versions.
```

Comment:
Should specify that old kits will still work if enabled _(for the time being)_

Reply:
Also, will this break the ability for kits to execute commands?

Reply:
Commands remain to work